status-im/status-react
# Discover - Implement recent statuses and statuses per popular tags #1977

fix for #1977

### Summary:
Discover face lifting initial tasks:

1. Recent statuses
2. Popular hashtags and statuses that contain them

### Steps to test:
- Open Status
- Switch to Discover tab
- See any discovered statuses or popular hashtags
- If the list is empty, swipe from left and post a new status or have a contact update status (make sure to include a hashtag in status body)

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready
